client.go

```diff
@@ -446,6 +450,18 @@ func DialWithTLS(network, address string, tlsConfig *tls.Config) (conn *Conn, er
 	return conn, nil
 }
 
+// Client.DialTimeoutEx acts like DialTimeout, but uses the client's settings.
```

Comment:
Where do you mean? In `DialTimeout` there is no access to `Client` or we have to add it as a parameter.